src/test/java/com/rapid7/client/dcerpc/mssamr/messages/Test_SamrQueryInformationDomain.java

```diff
@@ -57,6 +58,19 @@ public void SamrQueryLogOffInformationDomain() throws IOException {
         assertEquals(-9223372036854775808L, logOffInfo.getForceLogoff());
     }
 
+    @Test
+    public void SamrQueryLockoutInformationDomain() throws IOException {
+        String hexString = "000002000c00000000cc1dcffbffffff00cc1dcffbffffff0000";
+        SAMPRDomainLockoutInfo lockout = new SAMPRDomainLockoutInfo();
+        SamrQueryInformationDomainResponse<SAMPRDomainLockoutInfo> response = new SamrQueryInformationDomainResponse<>(
+            lockout, DomainInformationClass.DOMAIN_LOCKOUT_INFORMATION);
+        response.unmarshal(getPacketInput(hexString));
+
+        // 1800 seconds
+        assertEquals(-18000000000L, lockout.getLockoutDuration());
```

Comment:
Why is there a `-`

Reply:
@bli-r7 , FYI - https://github.com/rapid7/nexpose-r7shared/blob/9a73ebc4eb4752abb11592193fb065681886c080/src/main/java/com/rapid7/net/cifs/CifsUtil.java#L23-L59